tedana/tests/test_mapca.py

```diff
@@ -6,6 +6,50 @@
 import nibabel as nib
 from tedana import decomposition
 from pytest import raises
+from tedana.decomposition.ma_pca import _autocorr, _check_order, _parzen_win
+from tedana.decomposition.ma_pca import _subsampling, _kurtn, _icatb_svd, _eigensp_adj
+
+
+def test_autocorr():
+    """
+    Uni test on _autocorr function
+    """
+    test_data = np.array([1, 2, 3, 4])
+    test_result = np.array([30, 20, 11, 4])
+    autocorr = _autocorr(test_data)
+    assert np.array_equal(autocorr, test_result)
+
+
+def test_check_order():
+    """
+    Unit test on _check_order function
+    """
+    test_order = -1
+    with raises(ValueError) as errorinfo:
+        ord_out, w, trivwin = _check_order(test_order)
+    assert 'Order cannot be less than zero' in str(errorinfo.value)
+
+    test_order = 0
+    ord_out, w, trivwin = _check_order(test_order)
+    assert ord_out == test_order
+    assert trivwin
+
+    test_order = 1
+    ord_out, w, trivwin = _check_order(test_order)
+    assert ord_out == test_order
+    assert w == 1
+    assert trivwin
```

Comment:
Any way to separately test when `trivwin` is `False` ?

Reply:
Must have missed that one. Just added this. Thanks for catch @emdupre! 💯